### PR TITLE
Add support for deleting rows in sheet

### PR DESCRIPTION
**Pull request by amanda11**
Addresses https://github.com/StackStorm-Exchange/stackstorm-excel/issues/15 to add support to allow the user to delete rows by specifying the key of row they wish to delete.

Also added in unit-test for delete row, as well as a couple of tests on some of the existing actions, to check that the base mock class would also serve for testing of the existing classes.

### CODE REVIEW

**Comment by nmaludy** [comment]
@amanda11 can you ping me here when you're ready for a review?

**Comment by amanda11** [comment]
@nmaludy Hi - I think its all ready for review now.

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 0.2.6
```
Comment (nmaludy) [inline CHANGES.md]: small nit-pick here, could you bump to 0.3.0 since we're adding a new feature?

**Review by nmaludy — CHANGES_REQUESTED** [review]
One small change request for the version bump, otherwise looks great. Thank you so much for all of the tests!

**Review by nmaludy — APPROVED** [review]
LGTM Thank you SO MUCH for the PR!

**Pull request merged**